src/index.html

```diff
@@ -37,7 +37,8 @@
   <meta property="og:title" content="Just Arrived"/>
   <meta property="og:description" content="We give newcomers in Sweden a chance to enter the labor market"/>
   <meta property="og:image" content="/assets/images/logo.png"/>
-  <meta property="og:locale" content="sv"/>
+  <meta property="og:locale" content="sv_SE"/>
+  <meta property="fb:app_id" content="2287595228132792"/>
```

Comment:
Perhaps brake this out to a config-file or the likes. Might be hard to find later on,